File: task-3/Elixir/Sub-Task 3/main.exs
{n, _} = IO.gets("Enter rows: \n") |> Integer.parse

Enum.reduce(0..n-1, rem(n, 2), fn i, counter ->
    if counter == 0 do
        counter+2
    else
        stars = String.duplicate("*", counter)
        spaces = String.duplicate(" ", floor((n - counter) / 2))
        
        IO.puts(spaces <> stars <> spaces)
        
        if i < floor(n / 2), do: counter+2, else: counter-2
    end
end)

File: task-3/Ruby/Sub-Task 3/main.rb

puts  "Enter rows: "
n = gets.chomp.to_i
counter = n%2

for i in 0..(n-1)
	if counter == 0
		counter += 2
		next
	end

	stars = "*"*counter
	spaces = " "*(((n-counter)/2).floor())
	
	if i < (n/2).floor()
		counter += 2
	else
		counter -= 2
	end
	
	puts spaces+stars+spaces
end